web/src/helpers/translation.js

```diff
@@ -44,20 +44,20 @@ exports.getFullZoneName = function getFullZoneName(zoneCode) {
 
 exports.languageNames = [
   { shortName: 'ar', name: 'اللغة العربية الفصحى' },
-  { shortName: 'cs', name: 'čeština' },
-  { shortName: 'da', name: 'dansk' },
+  { shortName: 'cs', name: 'Čeština' },
+  { shortName: 'da', name: 'Dansk' },
   { shortName: 'de', name: 'Deutsch' },
   { shortName: 'en', name: 'English' },
-  { shortName: 'es', name: 'español' },
-  { shortName: 'fr', name: 'français' },
-  { shortName: 'it', name: 'italiano' },
+  { shortName: 'es', name: 'Español' },
+  { shortName: 'fr', name: 'Français' },
+  { shortName: 'it', name: 'Italiano' },
   { shortName: 'ja', name: '日本語' },
   { shortName: 'nl', name: 'Nederlands' },
-  { shortName: 'pl', name: 'polski' },
-  { shortName: 'pt-br', name: 'português (Brazilian)' },
+  { shortName: 'pl', name: 'Polski' },
```

Comment:
I can't speak for other languages, but I can tell you for sure this is wrong - we don't capitalize language names in Polish as is done in English or German.

Reply:
And why did you remove pt-br?

Reply:
pt-br is below, the display in Github is misleading because I attached my comment to the pl line